packages/neutrino/extensions.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   // Modifying a value here should have an accompanying change there as well.
   // We can't pull in neutrino there as that would potentially give us
   // conflicting versions in node_modules.
-  source: ['js', 'jsx', 'vue', 'ts', 'tsx', 'mjs'],
+  source: ['wasm', 'mjs', 'vue', 'jsx', 'tsx', 'ts', 'js'],
```

Comment:
Should we drop `.ts` and `.tsx` for now (here and for the eslint command), given no core Neutrino preset supports TypeScript, and the inclusion of those extensions apparently made things worse in #1044 (or at least caused confusion)?

Reply:
We can do this in a follow-up.